tests/shared/robot.spec.js

```diff
@@ -131,16 +131,16 @@ describe('Shared: Robot', function() {
         data: {
           message: 'foo',
           code: 'MYERROR',
-          stacktrace: error.stack
+          stack: error.stack
         }
       });
     });
 
     it('should handle a string error', function() {
```

Comment:
Fixed! I made the function handle strings anyway just to be on the safe side, since some node libraries throw strings (like UI Bootstrap). If this happens on the CLI writer, the current approach will swallow the error.